examples/fr2en.cpp

```diff
@@ -164,7 +164,8 @@ struct Model {
       precConfig.quantConfig.assertAllNodesQuantized = true;
     }
 
-    EE_.compile(F_, cctx);
+    EE_.compile(cctx);
+    bindings.allocate(EE_.getModule().getPlaceholders());
```

Comment:
Huh, this wasn't needed before? Wondering why it's needed now with EE2.

Reply:
Good catch, it's only only needed when training, removed.